### PR TITLE
Add tests for `seq`

### PR DESCRIPTION
**Pull request by c71n93**
Related to #2834

---

## PR-Codex overview
This PR focuses on adding tests for single dataization int equal to methods in `seq-tests.eo`.

### Detailed summary
- Added test for `seq-single-dataization-int-equal-to-test`
- Added test for `seq-single-dataization-int-equal-to-cache-problem-test`
- Defined sequences and operations for testing
- Checked equality and cache problems for single dataization with integers

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by yegor256** [comment]
@c71n93 what's up with this one? ready for review?

**Comment by yegor256** [comment]
@c71n93 @maxonfjvipon maybe this branch will work now, if you merge master into it?

**Comment by c71n93** [comment]
@maxonfjvipon check please

**Comment by c71n93** [comment]
@maxonfjvipon it's strange, qulice did not notice this unused imports. I will fix it.

File: eo-runtime/src/test/eo/org/eolang/seq-tests.eo [hunk 1]
```diff
@@ -127,3 +127,28 @@
...
+  eq. > @
+    seq
+      *
+        x.write 0
+        x.write (x.as-int.plus 1)
+        x.as-int
+    1
+
+# Test.
+[] > seq-single-dataization-int-equal-to-cache-problem-test
```
Comment (maxonfjvipon) [inline eo-runtime/src/test/eo/org/eolang/seq-tests.eo:143]: @c71n93 I believe this test should work, maybe try to enable it?

**Review by maxonfjvipon — APPROVED** [review]

**Comment by c71n93** [comment]
@yegor256 could you check this one, please?

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**